pyqmix_frontend/src/App.css

```diff
@@ -70,8 +70,8 @@ body {
     }
 }
 
-.text-modal {
-    white-space: pre-line;
+.modal_input {
```

Comment:
cool, but could you use `lowercase-with-dashes` as class names for the CSS? :) (I know, we use `lowercase_with_underscores` with Python – the web folks decided to handle things a little different though!)

Reply:
I actually did that, but then I got some weird error as if modal-input was a predefined class in the app, but could be I got fooled by the cache. But, long story short, yes, of course, will do.

Reply:
Interesting. Could you please paste the error message if this cannot be easily resolved?

Reply:
Yeah, probably just me fooling around ^^